scss/civicrm/administer/civi-case/_case-types.scss

```diff
@@ -34,10 +34,23 @@
       }
 
       .ui-tabs-panel {
+        display: table;
+        overflow-x: auto;
         padding: 0 !important;
+        width: 100%;
 
-        table:first-child {
+        // Makes sure any element inside the panel has a true width 100% depending
+        // on the widest sibling and not the panel's width:
+        > * {
```

Comment:
`*` should be used only if really necessary, is it the case here? You don't know what elements in particular you want to behave like a table row?

Reply:
I know the specific elements, but one of them is a div with no discerning class or attribute to use as a selector. I changed it to this:

```scss
> div,
> .crm-accordion-wrapper {
  // ...
}
```